SECURITY.md

```diff
@@ -1,43 +1,51 @@
 # Security Policy
-The Maintainers and contributors to KubeArmor take the security of our software seriously. 
+
+The Maintainers and contributors to KubeArmor take the security of our software seriously.
 The KubeArmor community has adopted the below security disclosures and response policy to promptly respond to critical issues.
 
 Please do not report security vulnerabilities through public GitHub issues.
 
 ## Security bulletins
+
 For information regarding the security of this project please join our [slack channel](https://join.slack.com/t/kubearmor/shared_invite/zt-1ltmqdbc6-rSHw~LM6MesZZasmP2hAcA/).
 
 ## Reporting a Vulnerability
+
 ### When you should?
+
 - You think you discovered a potential security vulnerability in KubeArmor.
 - You are unsure how a vulnerability affects KubeArmor.
 - You think you discovered a vulnerability in the dependency of KubeArmor. For those projects, please leverage their reporting policy.
 
 ### When you should not?
+
 - You need assistance in configuring KubeArmor for security - please discuss this is in the [slack channel](https://join.slack.com/t/kubearmor/shared_invite/zt-1ltmqdbc6-rSHw~LM6MesZZasmP2hAcA/).
 - You need help applying security-related updates.
 - Your issue is not security-related.
 
-### Please use the below process to report a vulnerability to the project:
-1. Email the **KubeArmor security group at support@accuknox.com**
+### Please use the below process to report a vulnerability to the project
+
+1. Email the **KubeArmor security group at <support@accuknox.com>**
 
-    * Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
-        * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
-        * Full paths of the source file(s) related to the manifestation of the issue
-        * Location of the affected source code (tag/branch/commit or direct URL) 
-        * Any special configuration required to reproduce the issue
-        * Step-by-step instructions to reproduce the issue
-        * Proof-of-concept or exploit code (if possible)
-        * Impact of the issue, including how an attacker might exploit the issue
+    - Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
```

Comment:
What's the difference between using `-` and `*` in md :smile: ? Because above you reversed the order.

Reply:
I think the `-` is a more conventional way to list items in Markdown.

Reply:
I see, let's stick to `-` only throughout the document then. Please reverse the above changes regarding the same.

Reply:
I think I have reverted the change. Please review. List items are prefixed with `-` at the moment.